src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -68,6 +68,68 @@ void appendStyled(FlutterLogEntry entry, String text) {
       abstract void render(FlutterLogEntry entry);
     }
 
+    abstract class Column extends ColumnInfo<DefaultMutableTreeNode, FlutterLogEntry> {
+      boolean show = true;
+      private TableCellRenderer renderer;
```

Comment:
add `@Nullable` maybe better for IDE lining.

Reply:
👍